### DISCUSSION TITLE
Random taps on hamster

### DISCUSSION THREAD

**Discussion by cmeept** [masterking32/MasterHamsterKombatBot#89] [Q&A] [answered]
Is it possible to make the taps on the hamster more random so as not to look like a bot? 3-4 taps per second for example or sleep after some clicks ?

**Comment by masterking32** [comment, marked as answer]
When you tap in-game, the game will send taps to the server when you stop tapping for example: when you have 10000 points and if you tap all of them non-stop, the client will send only one request to the server. that's similar to that.
